### PR TITLE
Ta 21 deployments

### PR DESCRIPTION
**Pull request by shanemacbride**
- Merge docker deployment ec2 instance terraform

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -31,11 +31,11 @@ pipeline {
           },
           jenkins_master: {
             dir('ansible-jenkins') {
-              git branch: 'master', url: 'https://github.com/liatrio/ansible-jenkins.git'
+              git branch: 'PA-21-deployments', url: 'https://github.com/liatrio/ansible-jenkins.git'
```
Comment (jphayes) [inline Jenkinsfile]: branch should be master, no?

File: Jenkinsfile [hunk 2]
```diff
@@ -31,11 +31,11 @@ pipeline {
...
             }
             sh "cp $JENKINS_HOME/hackathon_inventories/jenkins_master.inventory ansible-jenkins/inventory"
             sh "ansible-galaxy install liatrio.mount_persist_data"
-            sh "ansible-galaxy install geerlingguy.jenkins"
+            sh "ansible-galaxy install -f geerlingguy.jenkins"
```
Comment (jphayes) [inline Jenkinsfile]: what does `-f` do?

**Review by JGuzman4 — DISMISSED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (RobertKelly) [inline Jenkinsfile]: Is this supposed to be master?

**Review by jphayes — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (shanemacbride) [reply Jenkinsfile]: Answered in Slack 😄

File: Jenkinsfile (hunk 1 above)
Comment (shanemacbride) [reply Jenkinsfile]: Correct! Good catch, new commit incoming.

Comment (shanemacbride) [reply Jenkinsfile]: Correct! Good catch, new commit incoming.

File: Jenkinsfile [hunk 3]
```diff
@@ -72,30 +72,39 @@ pipeline {
...
+							withCredentials([sshUserPrivateKey(credentialsId: 'hackathon-key', keyFileVariable: 'keyFileVariable')]) {
+								sh "ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook --private-key $keyFileVariable -i  ansible-docker-nodes/inventory ./ansible-docker-nodes/docker_nodes.yml"
+							}
+          },
+          artifactory: {
+            dir('ansible-artifactory'){
+              git branch: 'master', url: 'https://github.com/liatrio/ansible-artifactory.git'
+            }
+            withCredentials([sshUserPrivateKey(credentialsId: 'hackathon-key', keyFileVariable: 'keyFileVariable')]) {
+              sh "echo hello"
```
Comment (RobertKelly) [inline Jenkinsfile]: Why `echo hello`

**Review by RobertKelly — COMMENTED** [review]
Was just curious

**Review by RobertKelly — APPROVED** [review]
Looks good - 5 Docker nodes to add

**Review by jphayes — APPROVED** [review]

**Pull request merged**